CONTRIBUTING.md

```diff
@@ -83,7 +83,30 @@ Flutter tests use [package:flutter_test](https://github.com/flutter/flutter/tree
  * `cd packages/flutter_tools`
  * `dart test/all.dart`
 
-`flutter test --flutter-repo` is a shortcut for those working on the flutter repository itself which runs all tests inside the `flutter` package regardless of the current working directory.
+Those working on the Flutter repo might find it useful to have a command-line alias for running Flutter's own tests
+regardless of the current working directory. For example, in your `.bashrc` on Linux, or `.profile` on Mac, define:
+
+```sh
+# Where you cloned the flutter repo
+export FLUTTER_REPO_DIR=/path/to/flutter/repository
```

Comment:
But shouldn't it point to the same place? I should have said, what about using `FLUTTER_ROOT` below instead of defining a new var?